bin/celery_worker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+LOGLEVEL=${CELERY_LOGLEVEL:-INFO}
+
+mkdir -p ../celerybeat
+
+echo "Starting celery worker"
+celery worker \
+    --app camunda_worker \
+    -l $LOGLEVEL \
+    --workdir src \
+    -B \
```

Comment:
Can you create a separate `celery_beat.sh` script? See https://docs.celeryproject.org/en/latest/reference/celery.bin.worker.html#cmdoption-celery-worker-b